src/index.js

```diff
@@ -37,6 +37,8 @@ export { Button                  } from './atoms/Button';
 export { BylineImage             } from './atoms/BylineImage';
 export { BylineName              } from './atoms/BylineName';
 export { BylineWithTwoLines      } from './atoms/BylineWithTwoLines';
+export { Description             } from './atoms/MainRecipe/Description';
+export { Container               } from './atoms/MainRecipe/youtubeFrame';
```

Comment:
Rename Container to YouTubeFrame, please. It's specific to YouTube because of the size of the padding.